redfish-core/lib/account_service.hpp

```diff
@@ -2048,6 +2048,22 @@ inline void requestAccountServiceRoutes(App& app)
                         messages::internalError(asyncResp->res);
                         return;
                     }
+                    
+                    // Create (modified) modGroupsList from allGroupsList that
+                    // does not contain the ipmi group.
+                    std::vector<std::string> modGroupsList;
+
+                    for (unsigned int index = 0; index < allGroupsList->size();
+                                                                 index++)
+                    {
+				    	if (allGroupsList->at(index) != "ipmi") 
```

Comment:
Nit: Does this pass the code format check?  This is indented with tabs, and it needs to be spaces.